src/main/java/uk/gov/cshr/controller/LoginController.java

```diff
@@ -18,10 +18,10 @@
 
     @RequestMapping("/login")
     public String login( HttpServletRequest request, HttpServletResponse response) throws IOException {
-        DefaultSavedRequest dsr = (DefaultSavedRequest) request.getSession().getAttribute("SPRING_SECURITY_SAVED_REQUEST");
-        if (dsr != null && dsr.getQueryString() == null) {
-            response.sendRedirect(lpgUiUrl);
-        }
+//        DefaultSavedRequest dsr = (DefaultSavedRequest) request.getSession().getAttribute("SPRING_SECURITY_SAVED_REQUEST");
```

Comment:
BUG - Why is this commented out?

Reply:
Because if you request the account/password page and you're not logged in you're redirected to UI rather than the login page.

Reply:
That is fine, you shouldn't be on that page if you aren't logging in anyways